### ISSUE TITLE
[SIGIR2019] Learning More From Less: Towards Strengthening Weak Supervision for Ad-Hoc Retrieval

### ISSUE THREAD

**Issue by mocobeta** [mocobeta/paper-reading-records#2]
## URL: 論文URL

https://arxiv.org/pdf/1907.08657.pdf

## Author: 著者

Dany Haddad (University of Texas at Austin) , Joydeep Ghosh (University of Texas at Austin) 

## Motivation: 研究の出発点 (課題は何？)

Recent works have shown the potential of leveraging unsupervised methods as sources of weak supervision for the retrieval task. However, they require training on as many as 10^13 training examples to surpass the performance of the unsupervised baseline. 

## Method: 研究手法 (どんなシステム/アルゴリズムを作った？どんな実験をした？)

They proposed two models:
- Noise-aware model that does not require ground truth labels, with soft relevance labels 
- Influence-aware model that identify examples that hurt the generalization performance and eliminate them from the training set
They evaluate their methods to ad-hoc retrieval on the Robust04 corpus.

## Insight: 結果と知見 (何が得られた？どんな条件下でうまくいく？うまくいかない場合はどんな時？)

Proposed methods outperform two baselines, the unsupervised ranker (QL) and the _rank_ model.

## Keyword: 新しいキーワードはある？

- weak supervision for the retrieval task
- soft relevance label

## Unknown: 残った課題は？

- For the nose-aware model, learn a better joint distribution
- For the influence-aware model, use the softrank loss rather than cross-entropy and instead compute set influence rather than the influence of a single training example

## Reflection: 考察 

- soft relevance label は直感と合う。
- feed-forward 時の computational cost は？

## Reference: 関連研究 (次に読むのは？)

Alexander Ratner, Stephen H Bach, Henry Ehrenberg, Jason Fries, Sen Wu, andChristopher Ré. 2017. Snorkel: rapid training data creation with weak supervision. Proceedings of the VLDB Endowment11, 3 (Nov.2017), 269–282
